pkg/client/cluster-essentials/clusterinit.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package clusteressentials has APIs for bootstrapping kubernetes cluster
+package clusteressentials
```

Comment:
I think you probably misunderstood my previous comment. We want a directory structure like this:

```
cluster-essentials
├── go.mod
└── pkg
    ├── client
    │   ├── fake.go
    │   ├── helpers.go
    │   ├── install.go
    │   └── install_test.go
    └── testdata
```